shepherd_bms/include/bmsConfig.h

```diff
@@ -1,16 +1,37 @@
-#ifndef BMS_COMFIG_H
+#ifndef BMS_CONFIG_H
 #define BMS_CONFIG_H
 
+// Hardware definition
 #define NUM_SEGMENTS        4
 #define NUM_CHIPS           NUM_SEGMENTS*2
 #define NUM_CELLS_PER_CHIP  9
 #define NUM_THERMS_PER_CHIP 32
 
-// Hardware limits
+// Firmware limits
 #define MAX_TEMP            65 //degrees C
-#define MIN_TEMP            -5 // deg C
-
+#define MIN_TEMP            -15 // deg C
 #define MAX_VOLT_MEAS       65535
 #define MIN_VOLT_MEAS       0
 
+// Boosting Parameters
+#define BOOST_TIME          5   // seconds
+#define BOOST_RECHARGE_TIME 30  // seconds
+#define CONTDCL_MULTIPLIER  1.01
```

Comment:
```suggestion
#define CONTDCL_MULTIPLIER  2
```
Should be updated while I remember